frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.styled.tsx

```diff
@@ -17,6 +18,14 @@ export const ActionSubtitle = styled.span`
   margin-top: 4px;
 `;
 
+export const ActionSubtitlePart = styled.span`
+  &:not(:last-child)::after {
+    content: "·";
+    margin-left: 6px;
```

Comment:
Uses the exact same space as in our Figma file.

#### Figma
<img width="678" alt="image" src="https://user-images.githubusercontent.com/1937582/216436646-7407c17a-6613-43c6-9def-ce897e14f02f.png">